src/cmd/provisioner/run.go

```diff
@@ -79,6 +80,11 @@ func (r *Run) Execute(ctx context.Context, f *flag.FlagSet, args ...interface{})
 		return subcommands.ExitFailure
 	}
 	if err := provisioner.Run(ctx, deps, *stateDir, c); err != nil {
+		if errors.Is(err, provisioner.ErrRebootRequired) {
+			log.Println("Reboot is required to continue provisioning. Please reboot the system and resume provisioning with the `resume` subcommand.")
```

Comment:
will this log message be the same as resume.go:57? Might be better to share it in that case

Reply:
Done